stratum/templates/ds.yaml

```diff
@@ -20,22 +20,17 @@ spec:
         {{- toYaml . | nindent 8 }}
       {{- end }}
       restartPolicy: Always
-      initContainers:
-        - name: init-huge
-          image: ubuntu:16.04
-          securityContext:
-            privileged: true
-          command: ['sh', '-c', "sysctl vm.nr_hugepages && sysctl vm.nr_hugepages=128"]
+      hostNetwork: true
       containers:
       - name: stratum-bf
         image: {{ .Values.registry }}/{{ .Values.image }}:{{ .Values.tag }}
         command: ["/usr/bin/start-stratum.sh"]
         args: ["--bf_sim"]
-        env:
-        - name: PLATFORM
-          value: x86-64-accton-wedge100bf-65x-r0
         securityContext:
           privileged: true
+          sysctls:
+          - name: vm.nr_hugepages
+            value: "128"
```

Comment:
@hwchiu This seems working fine. I don't know why we wrote an initContainer previously 🤔

Reply:
👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍